Kinvey/KinveyApp/PerformanceTestData.swift

```diff
@@ -14,7 +14,7 @@ class PerformanceTestData: UIViewController {
     let client = Client(appKey: "kid_b1d6IY_x7l", appSecret: "079412ee99f4485d85e6e362fb987de8")
     
     func store<T: Persistable>() -> DataStore<T> where T: NSObject {
-        return DataStore<T>.collection(.network, client: self.client)
+        return DataStore<T>.collection(.network, options: Options(client: self.client))
```

Comment:
Is this a sample app? Did a new warning appear due to an API change made sometime earlier to the SDK?

Reply:
yes, this is not necessary related to Swift 4.1, but a deprecated usage for a while ago